Game/GameSinglePlayerScene.swift

```diff
@@ -38,10 +38,24 @@ class GameSinglePlayerScene: SKScene {
 	// MARK: - Events
 	
 	override func keyDown(with event: NSEvent) {
-//		switch event.keyCode {
-//		default:
+		//hardcoded keys for now...
+		switch event.keyCode {
+		case Keycode.leftArrow:
+			game.localPlayerInputMap[.moveLeft].activated = true
+		case Keycode.rightArrow:
+			game.localPlayerInputMap[.moveRight].activated = true
+		case Keycode.downArrow:
+			game.localPlayerInputMap[.moveDown].activated = true
+		case 0x00://US Keyboard 'a'
```

Comment:
Well smack me around and call me Suzy...

I went and read things and convinced myself this was not the case. I went to go find such things and have found I'm wrong. Sigh.